src/sdk/config_fs.js

```diff
@@ -839,6 +849,20 @@ class ConfigFS {
         return { parsed_account_data, string_account_data };
     }
 
+    /**
+     * remove_encrypted_secret_key will remove the encrypted_secret_key property from an identity
+     * since we allow to have more than one access key, we hold the index (but most cases are with 1)
+     * @param {object} config_data
```

Comment:
please add returns, it helps the linter find issues

Reply:
Added, thanks.